Components/Inputs/input.css

```diff
@@ -1,16 +1,15 @@
-@import "../../Colors/colors.css";
+@import "../../Variables/variables.css";
 
-div>* {
-    display: block;
-    padding: .5rem;
-    width: 20rem;
+div > * {
+  display: block;
+  padding: 0.5rem;
+  width: 20rem;
```

Comment:
Don't give generic style, It might cause problem in future.

Reply:
Yes, thanks for pointing it out, I've updated.